config/checkstyle_checks.xml

```diff
@@ -55,6 +55,11 @@
     <property name="offCommentFormat" value="CSOFF\: .*"/>
     <property name="onCommentFormat" value="CSON\: .*"/>
   </module>
+  <module name="SuppressWithNearbyTextFilter">
+    <property name="nearbyTextFormat"
```

Comment:
>format

Sounds very weird.
What if name it nearbyTextPattern ? At least it will be clear it is regexp.

Reply:
Then I will have to change all of the others to `...Pattern` because right now they are `...Format`. On it.

Reply:
Done.